### PR TITLE
ProductCode/ARP: ActiveState.ActivePerl version 5.28.1001

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23761)

### CODE REVIEW

**Comment by vedantmgoyal2009** [comment]
Is it because the URL is a vanity URL?

File: manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.installer.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ Installers:
   Scope: machine
   InstallerUrl: https://cli-msi.s3.amazonaws.com/ActivePerl-5.28.msi
   InstallerSha256: 3481AC53B97038F4A84FFC4BA39BE3CD54A988B4F172CCBB6C429CDEE32D1A55
```
Comment (ItzLevvie) [inline manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.installer.yaml:16]:
```suggestion
  InstallerSha256: bae2f2be63f1a713b8c0965c07d383bebc06096808ecd4d6d8235a3a15350c1e
```

File: manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.installer.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Automatically updated by the winget bot at 2021/Jul/30
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 PackageIdentifier: ActiveState.ActivePerl
-PackageVersion: 5.28.1001
+PackageVersion: 5.28.1001.0
```
Comment (ItzLevvie) [inline manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.installer.yaml:4]:
```suggestion
PackageVersion: 5.28.1001
```

File: manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.locale.en-US.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Automatically updated by the winget bot at 2021/Jul/30
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
 PackageIdentifier: ActiveState.ActivePerl
-PackageVersion: 5.28.1001
+PackageVersion: 5.28.1001.0
```
Comment (ItzLevvie) [inline manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 5.28.1001
```

File: manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Automatically updated by the winget bot at 2021/Jul/30
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
 PackageIdentifier: ActiveState.ActivePerl
-PackageVersion: 5.28.1001
+PackageVersion: 5.28.1001.0
```
Comment (ItzLevvie) [inline manifests/a/ActiveState/ActivePerl/5.28.1001.0/ActiveState.ActivePerl.yaml:4]:
```suggestion
PackageVersion: 5.28.1001
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**